packages/react-components/src/components/Picker/PickerList.module.scss

```diff
@@ -0,0 +1,72 @@
+$base-class: 'picker-list';
```

Comment:
Have we abandoned 'lc-something' convention?

Reply:
`lc-` prefix will be added in build step.

You can find the configuration for this here:

```
//vite.config.js

  css: {
    modules: {
      generateScopedName: 'lc-[name]__[local]___[hash:base64:5]',
    },
  },
```